client/src/containers/ChannelMembers/ChannelMembers.js

```diff
@@ -7,8 +7,8 @@ import callApi from "../../libs/axios";
 import { fetchMembersData } from "../../containers/Workspace/utils";
 import { useAppContext } from "../../containers/App/AppContext";
 import AddMember from "./AddMember";
-import isEqual from "lodash/isEqual";
-import uniq from "lodash/uniq";
+import isEqual from "lodash-es/isEqual";
```

Comment:
You can import form single source like 
`import { isEqual, uniq } from "lodash-es"